labs/lab6/lab6.adoc

```diff
@@ -98,13 +41,15 @@ If the web session has not been retained from a prior lab, login with the follow
 Username **admin** +
 Password **rhte2017**
 
-After logging in, navigate to the **Templates** page and locate the **1-Provision-and-Scale-OpenShift** workflow job template. Click the `rocket` icon to start the job.
+After logging in, navigate to the **TEMPLATES** page and locate the **2-Scaleup_OpenShift_on_AWS** workflow job template. Click the **RocketShip** icon to start the job.
```

Comment:
I would have them SSH to the master and do `oc get nodes` before scaleup and after.

Reply:
Time is a factor. Would want them to kick off the job asap. Could revisit later on whether we can have them run the command just after triggering job as it shouldnt have added them quite yet